src/components/LanguageInput.vue

```diff
@@ -79,11 +79,11 @@ export default {
 </template>
 
 <style lang="scss">
-@import '@wmde/wikit-tokens/variables';
+@import '@wikimedia/codex-design-tokens/theme-wikimedia-ui';
 
 /* stylelint-disable selector-class-pattern */
 .wbl-snl-language-lookup .wikit .wikit-Lookup__label-wrapper {
-	gap: $dimension-spacing-xsmall;
+	gap: $spacing-50;
```

Comment:
This increases the gap from `.25em` (normally = `4px`) to `8px`. According to T369510 that’s intended. Likewise in `LemmaInput`, `LexicalCategoryInput`, `RequiredAsterisk`, `SpellingVariantInput`.